release_notes/Service_Fabric_ReleaseNotes_80.md

```diff
@@ -0,0 +1,113 @@
+|Java SDK  |Java for Linux SDK  | 1.0.5 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 8.0.0 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 8.0 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current And Upcoming Breaking Changes](#Current-And-Upcoming-Breaking-Changes)
+* [Service Fabric Runtime](#service-fabric-runtime)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Annoucements
+* Ability to move stateless service instances
+* Ability to add parameterized DefaultLoad in the application manifest
+* For singleton replica upgrades - ability to have some of the cluster level settings to be defined at an application level
+* Ability for smart placement based on node tags
+* Ability to define percentage threshold of unhealthy nodes that influence cluster health
+* Ability to query top loaded services
+* Ability to add a new interval for new error codes 
+* Capability to mark service instance as completed
+* Enable support for .NET 5 for Windows
+* Added readiness probe for containerized applications
+* Enable UseSeparateSecondaryMoveCost to true by default
+* Fixed StateManager to release the reference as soon as safe to release
+
+
+## Current And Upcoming Breaking Changes
+Guest executable and container applications created or upgraded in SF clusters with runtime versions 7.1+ are incompatible with prior SF runtime versions (e.g. SF 7.0).<br/>
+Following scenarios are impacted:<br/>
+* An application with guest executables or containers is created or upgraded in an SF 7.1+ cluster.<br/>
+The cluster is then downgraded to a previous SF runtime version (e.g. SF 7.0).<br/>
+The application fails to activate (see symptoms).<br/>
+* A cluster upgrade from pre-SF 7.1 version to SF 7.1+ version is in progress.<br/>
+In parallel with the SF runtime upgrade, an application with guest executables or containers is created or upgraded.<br/>
+The SF runtime upgrade starts rolling back (due to any reason) to the pre-SF 7.1 version.<br/>
+The application fails to activate (see symptoms).<br/>
+
+To avoid issues when upgrading from a pre-SF 7.1 runtime version to an SF 7.1+ runtime version, do not create or upgrade applications with guest executables or containers while the SF runtime upgrade is in progress.<br/>
+* The simplest mitigation, when possible, is to delete and recreate the application in SF 7.0.<br/>
+* The other option, is to upgrade the application in SF 7.0 (for example, with a version only change).<br/>
+
+If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+
+
+
+## Service Fabric Runtime
+
+| Versions | IssueType | Description | Resolution | 
```

Comment:
We should also include support for the wave based upgrade model. You can find the documentation here: https://github.com/MicrosoftDocs/azure-docs-pr/pull/151266

Reply:
Added. Please review.